Rust toolchain is outdated

**Describe the bug**

Build blaze with default rust toolchain fails

```
[ERROR] [main] error: rustc 1.81.0-nightly is not supported by the following package:
[ERROR] [main]   poem@3.1.7 requires rustc 1.83
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -16,5 +16,5 @@
 # under the License.
 
 [toolchain]
-channel = "nightly-2024-06-27"
+channel = "nightly"
 components = ["rust-src", "cargo", "rustfmt", "clippy"]

